src/theories/HigherCategory.jl

```diff
@@ -21,26 +20,26 @@ abstract type CellExpr{T} <: CategoryExpr{T} end
 
 """ Theory of *2-categories*
 """
-@signature ThCategory2{Ob,Hom,Hom2} <: ThCategory{Ob,Hom} begin
-  """ 2-morphism in a 2-category """
-  Hom2(dom::Hom(A,B), codom::Hom(A,B))::TYPE ⊣ (A::Ob, B::Ob)
+@signature ThCategory2 <: ThCategory begin
+  # """ 2-morphism in a 2-category """
```

Comment:
Ditto for the docstrings: we should be able to capture docstrings, so let's make sure to have an issue tracking it if we can't do that yet.